### PR TITLE
Add tags for SQS

### PR DESCRIPTION
**Pull request by ken9ross**
## Intent of Change

- New feature (non-breaking change which adds functionality)

## Description
Include tags for SQS components

## Motivation and Context
https://github.com/hamlet-io/engine/issues/1133

## How Has This Been Tested?
Locally

### Prerequisite PRs:

- None

### Dependent PRs:

- None

### Consumer Actions:

- None

### CODE REVIEW

File: aws/components/sqs/setup.ftl [hunk 1]
```diff
@@ -71,6 +71,10 @@
                 name=dlqName
                 retention=1209600
                 receiveWait=20
+                tags=
+                    getOccurrenceCoreTags(
+                        occurrence,
+                        formatComponentFullName(core.Tier, core.Component, dlqName))
```
Comment (roleyfoley) [inline aws/components/sqs/setup.ftl:77]: Might as well keep the name the same as the one used above to name the resource

```suggestion
                        dlqName
```

File: aws/components/sqs/setup.ftl [hunk 2]
```diff
@@ -89,6 +93,10 @@
                 solution.DeadLetterQueue.MaxReceives,
                 solution.DeadLetterQueue.MaxReceives > 0,
                 (environmentObject.Operations.DeadLetterQueue.MaxReceives)!3)
+            tags=
+                getOccurrenceCoreTags(
+                    occurrence,
+                    formatComponentFullName(core.Tier, core.Component, sqsName))
```
Comment (roleyfoley) [inline aws/components/sqs/setup.ftl:99]:
```suggestion
                    sqsName
```

Same as above I think those two values end up being the same

**Review by roleyfoley — CHANGES_REQUESTED** [review]

File: aws/components/sqs/setup.ftl (hunk 1 above)
Comment (ken9ross) [reply aws/components/sqs/setup.ftl]: Cool, done

File: aws/components/sqs/setup.ftl (hunk 2 above)
Comment (ken9ross) [reply aws/components/sqs/setup.ftl]: Cool, done

File: awstest/modules/sqs/module.ftl [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                        "sqsbase" : {
+                            "sqs" : {
+                                "Profiles" : {
+                                    "Testing" : [ "sqsbase" ]
+                                },
+                                "deployment:Unit" : "aws-sqs-base",
+                                "Instances" : {
+                                    "default" : {}
+                                },
+                                "Ordering" : "FirstInFirstOut"
```
Comment (roleyfoley) [inline awstest/modules/sqs/module.ftl:27]: I don't think that is required as a base level option. FIFO is an optional service

**Review by roleyfoley — CHANGES_REQUESTED** [review]

**Comment by ml019** [comment]
@ken9ross suggest prepending "Fixes " to the issue reference in the PR description - that way the issue is closed when the PR is merged. Its a github feature. There is a range of keywords you can use - ask Dr Google if curious.

**Comment by ken9ross** [comment]
@ml019 I wasn't sure if I should be using "fixes" as this is only in the aws component and not azure (and the initial ticket was in engine - which I'm interpreting as being for all supported engines ie aws and azure). Should I be using "fixes" in this case? ie Are we happy to close an issue if it's fixed for aws and not azure?

File: awstest/modules/sqs/module.ftl (hunk 3 above)
Comment (ken9ross) [reply awstest/modules/sqs/module.ftl]: Done

**Review by roleyfoley — APPROVED** [review]

**Pull request merged**